src/models/daily-auths-report-data.test.ts

```diff
@@ -25,7 +25,7 @@ describe("DailyAuthsReportData", () => {
         yearMonthDayParse("2021-01-01"),
         yearMonthDayParse("2021-01-03"),
         "local",
-        fetch
+        fetch as typeof window.fetch
```

Comment:
fetch-mock's types seemed to require a `URL` but we were passing in `string` so this was a quick way to tell it to be quiet 😬 

I tried updating to send a URL in the code, but then that threw with a few "invalid URL" errors

Reply:
Yeah, I think the `fetch-mock` types are incorrect because the mocked function signature doesn't allow for `URL` objects, which are valid in `fetch` global.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e7c664fcae5f8116097f01bb05d7b2d6d2cb3dde/types/fetch-mock/index.d.ts#L626

(FYI, was there a reason we use DefinitelyTyped for this package's types, rather than the [package's first-party types](https://github.com/wheresrhys/fetch-mock/tree/master/types)? From some initial testing, it won't help with this specific problem though)

Reply:
Looks like we can shrink to at least this:

```json
{
  "root": true,
  "extends": ["plugin:@18f/eslint-plugin-identity/recommended"],
  "plugins": ["@18f/eslint-plugin-identity"],
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": 2021
  },
  "env": {
    "node": true,
    "browser": true
  },
  "rules": {
    "jsx-a11y/label-has-associated-control": ["error", { "assert": "either" }],
    "no-shadow": "off",
    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.*", "./*"] }],
    "react/no-unknown-property": [
      "error",
      { "ignore": ["font-size", "stroke-width", "text-anchor"] }
    ]
  },
  "settings": {
    "react": {
      "pragma": "h",
      "fragment": "Fragment",
      "version": "16.2"
    }
  },
  "ignorePatterns": ["_site/**"]
}
```

Reply:
I think it was just habit of mine to install them that way... removed in d970c2eff25e33b29d34a8e475ce75f6a53ee4e6